# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/ProxyAutoConfig.cpp
# Commit: 48eaa84c206d
# Full Hash: 48eaa84c206dbdda637c57b55ddbd07e975ba103
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1637500
# File Overlap Count: 2
# Description:
#   Bug 1637500 - P2. Rename methods as they are not always dealing with "threads". r=froydnj
#   
#   Before P1, GetCurrentThreadSerialEventTarget would have always returned the same data as NS_GetCurrentThread, making the comment incorrect Now it will properly return the running TaskQueue if any.
#   
#   This change of name more clearly exposes what they are doing, as we aren't always dealing with threads directly; but a nsISerialEventTarget
# ==============================================================================

diff -r 00687d06a8a6 -r 48eaa84c206d netwerk/base/ProxyAutoConfig.cpp
--- a/netwerk/base/ProxyAutoConfig.cpp	Tue Jun 23 05:02:39 2020 +0000
+++ b/netwerk/base/ProxyAutoConfig.cpp	Tue Jun 23 05:05:36 2020 +0000
@@ -449,7 +449,7 @@
       nsIDNSService::GetFlagsFromTRRMode(nsIRequest::TRR_DISABLED_MODE);
 
   if (NS_FAILED(dns->AsyncResolveNative(aHostName, flags, helper,
-                                        GetCurrentThreadEventTarget(), attrs,
+                                        GetCurrentEventTarget(), attrs,
                                         getter_AddRefs(helper->mRequest)))) {
     return false;
   }